Orange/widgets/visualize/tests/test_owscatterplot.py

```diff
@@ -102,3 +103,26 @@ def test_report_on_empty(self):
         self.widget.report_plot.assert_not_called()
         self.widget.report_caption.assert_not_called()
         self.widget.report_items.assert_not_called()
+
+    def test_data_column_nans(self):
+        """
+        ValueError cannot convert float NaN to integer.
+        In case when all column values are NaN then it throws that error.
+        GH-2061
+        """
+        dat = """\
```

Comment:
Is there a special reason why the dataset is generated in this way? (and not with table like in https://github.com/biolab/orange3/pull/2057/files#diff-1a66c8eae521f07f1c5d904e0cd025a7R117)?